redaxo/src/addons/phpmailer/lib/mailer.php

```diff
@@ -45,7 +45,7 @@ public function __construct($exceptions = false)
             $this->addBCC($bcc);
         }
 
-        $this->log = $addon->getConfig('log');
+        $this->log = $addon->getConfig('mail_archive');
```

Comment:
Die Property dann besser auch umbenennen in `$this->archive` oder `$this->mailArchive` (wobei eigentlich klar ist, dass es um die Mails geht)

Reply:
erledigt